Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -107,6 +116,19 @@ public async Task RemoveResponse(DiscordRequest request, Contexts contexts)
         [DiscordCommand("responses")]
         public async Task Responses(DiscordRequest request, Contexts contexts)
         {
+            // do testu obsługi reakcji:
+            if (!IsReactionSupportDelegateAdded)
```

Comment:
to powinien ogarniać serwis a nie kontroler 
dla kontrolera to nadmiarowy kod

co jak ktoś o tym zapomni?

Reply:
Można powiedzieć że `to jego problem że zapomniał, powinien pamiętać`

Ale ostatecznie to nasz problem, bo coś co powinno działać nie działa

Im mniej miejsc w których można zrobić "głupi błąd" tym lepiej